experiments/aws/parallel-cluster/cpu/experiment/results/32-node/amg2023/amg-32n-20-iter-1.err

```diff
@@ -0,0 +1,20 @@
+--------------------------------------------------------------------------
```

Comment:
Note for all runs - you can delete the `.gitkeep`s if you like! But no harm to have.